### DISCUSSION TITLE
I want to generate a slender female figure, but it keeps generating fuller figures

### DISCUSSION THREAD

**Discussion by grayistanbul** [lllyasviel/Fooocus#2508] [Q&A] [answered]
Here is my prompt: I provided four reference images for Fooocus to reference, but it consistently generates fuller-bodied women, and the skin tone is not the super fair complexion I desire. Is there an issue with the prompt?

Japanese Office lady,A curvaceous figure with a height of 167 cm and a weight of 42 kg,Oval faces,Super fair skin,A waist-to-hip ratio of around 36:24:36,A body fat percentage of 18%,Visible collarbones,slender arms and thighs,ultra details,beautiful,Wearing a mask on the face,White bodycon slit tight skirt,Eye of Detail, The sun's natural light shining on the body, prominent veins visible on the breasts,Photography: "Captured with a 50mm lens, this photograph of a 25-year-old girl exudes a sense of serene contemplation. Inspired by the works of Japanese photographer Rinko Kawauchi, the soft focus and muted colors of the image convey a sense of introspection. " --stylize 1000

The reference images I provided
![355437_0](https://github.com/lllyasviel/Fooocus/assets/162002014/5def08d2-2b21-45f5-a219-d4308724868a)
![355463_0](https://github.com/lllyasviel/Fooocus/assets/162002014/1e23003d-5b94-4b72-8c87-ff031c8d5b7b)
![355454_0](https://github.com/lllyasviel/Fooocus/assets/162002014/a6395e2c-0be0-4151-a652-21673296eab3)
![355467_0](https://github.com/lllyasviel/Fooocus/assets/162002014/0d8bd9eb-3a42-4192-b34e-6c49f791cced)

AI fuller-bodied women
![2024-03-11_04-28-54_3124](https://github.com/lllyasviel/Fooocus/assets/162002014/cee748a9-b3ce-4417-bdaf-a289ca9c5c61)
 ![2024-03-11_04-31-09_2601](https://github.com/lllyasviel/Fooocus/assets/162002014/cbd1558d-316d-44fe-aad8-a9b2efbc8f12).

Setting info:
![螢幕擷取畫面 2024-03-11 081118](https://github.com/lllyasviel/Fooocus/assets/162002014/d0ce8c16-356c-4449-bf0d-dac640fce2d7)
![螢幕擷取畫面 2024-03-11 081132](https://github.com/lllyasviel/Fooocus/assets/162002014/bded810a-090d-4b3a-ad04-65db772d81d0)
![螢幕擷取畫面 2024-03-11 083502](https://github.com/lllyasviel/Fooocus/assets/162002014/aebb533d-4eaa-4ba6-96f0-7bf023b3d36a)

**Comment by ILikeToasters** [comment, marked as answer]
Don't have time to go into to much detail but will mention that your prompt is probably too complex and really messing with things. Start simple and work your way up. It is not going to know much of the stuff in that prompt. Measurements and such will not do anything. The word fat in the prompt will likely influence it in a way you do not want. Keep to simple words like thin and slender. Use prompt weights like (slender:1.3) to increase the impact those words have. Also make use of the negative prompt for things you don't want. It doesn't have a major impact but will help with some things.

**Reply by grayistanbul** [reply]
thank you

**Comment by xhoxye** [comment]
You may have misunderstood the curvaceous figure?

**Comment by bblackmoor** [comment]
Try "athletic" and "fit".
